src/types/orchestrationTypes.ts

```diff
@@ -1,13 +1,18 @@
 import { FunctionOptions, FunctionTrigger, InvocationContext } from "@azure/functions";
-import { TokenSource } from ".";
+import { Task, TokenSource } from ".";
 import { DurableOrchestrationContext } from "../classes";
 
 /**
  * Type of a Generator that can be registered as an orchestration
+ * @param TReturn the type of the return value of the orchestration
  */
-export type OrchestrationHandler = (
+export type OrchestrationHandler<TReturn = any> = (
     context: OrchestrationContext
-) => Generator<unknown, unknown, any>;
+) => Generator<
+    Task, // orchestrations yield Task types
```

Comment:
nitpick: can we add a "only" yields Task types?
```suggestion
    Task, // orchestrations yield only Task types
```

Reply:
Done, thanks :)